ktor-utils/jvm/src/io/ktor/util/EncodersJvm.kt

```diff
@@ -48,11 +48,32 @@ private fun CoroutineScope.inflate(
         val header = source.readPacket(GZIP_HEADER_SIZE)
         val magic = header.readShortLittleEndian()
         val format = header.readByte()
-        val padding = header.readBytes()
+        val flags = header.readByte().toInt()
+        // val time = header.readInt()
+        // val extraFlags = header.readByte()
+        // val osType = header.readByte()
```

Comment:
Why are the commented out variables necessary? May be worth either deleting them or using them.

Reply:
I added an extra comment, but I would keep them for reference or future usage.